src/media/css/normalize.css

```diff
@@ -0,0 +1,424 @@
+/*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
```

Comment:
Yes, this question also came across to me too. So do you recommend removing this?